app/components/mig/menu/menu.styl

```diff
@@ -1,2 +1,30 @@
-button
-  border 10px solid gold
+$mantis = #3cb9a0
+
+h1
+  text-align center
+  font-size 0
+  margin 30px 0
+
+  svg
+    midth 150px
+    max-width 50%
+
+nav
+  text-align center
+
+  button
```

Comment:
Because multiple buttons are grouped and I wanted a nice way to put them in
a block element. Otherwise they kind of float at the top level of the
domfragment.

Ok flying noooooow...

On Friday, July 3, 2015, Ed Moore <notifications@github.com> wrote:

> In app/components/mig/menu/menu.styl
> <https://github.com/migme/surfboard/pull/29#discussion_r33849050>:
>
> > -  border 10px solid gold
> > +$mantis = #3cb9a0
> > +
> > +h1
> > +  text-align center
> > +  font-size 0
> > +  margin 30px 0
> > +
> > +  svg
> > +    midth 150px
> > +    max-width 50%
> > +
> > +nav
> > +  text-align center
> > +
> > +  button
>
> If it's a web component, you shouldn't need to embed button in nav
>
> nav button {}
>
> could just be
>
> button {}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/migme/surfboard/pull/29/files#r33849050>.
>